website/docs/docs/cloud/use-dbt-copilot.md

```diff
@@ -30,10 +30,14 @@ This page explains how to use <Constant name="copilot" /> to:
 
 <CopilotEditCode/>
 
+## Build visual models
```

Comment:
added headers in here as the removal of them broke the links in lines 22 and 23 above.